src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -466,6 +466,41 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
         }
 
         [Fact]
+        public async Task GetAsync_ResponseWithSetCookieHeaders_AllCookiesRead()
+        {
+            // Cookie names/values, with one of the values deliberately over 128 characters,
+            // which is the size of WinHttpHandler's initial response header buffer size, to
```

Comment:
> Is there any buffer size issues that CurlHandler has?

Unlikely. Curl just hands us a pointer to a native memory, that we read. We don't have to allocate anything like with WinHttpHandler.